packages/components/src/FormField/FormField.tsx

```diff
@@ -198,12 +198,16 @@ export function FormField(props: FormFieldProps) {
             onControllerBlur();
           }
 
-          function setAutocomplete(autocompleteSetting: boolean | string) {
-            if (autocompleteSetting === "one-time-code") {
-              return "one-time-code";
+          function setAutocomplete(
+            autocompleteSetting: boolean | AutocompleteTypes,
```

Comment:
Instead of doing this, i **think** that you can do `FormFieldProps["autocomplete"]`. In that case you dont need to export/import the `AutocompleteTypes`

Reply:
done!